sdk/typescript/src/cryptography/secp256k1-publickey.ts

```diff
@@ -70,6 +71,6 @@ export class Secp256k1PublicKey implements PublicKey {
     let tmp = new Uint8Array(SECP256K1_PUBLIC_KEY_SIZE + 1);
     tmp.set([SIGNATURE_SCHEME_TO_FLAG['Secp256k1']]);
     tmp.set(this.toBytes(), 1);
-    return sha3.sha3_256(tmp).slice(0, 40);
+    return sha3.sha3_256(tmp).slice(0, SUI_ADDRESS_LENGTH * 2);
```

Comment:
I'd a comment here that this double size is because of hex being 100% longer (each hex char represents half a byte of information)